### PR TITLE
Fix 80 warnings coming from gcc8.2

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

**Comment by radare** [comment]
fixes https://github.com/radare/radare2/issues/12213

File: shlr/tcc/tccgen.c [hunk 1]
```diff
@@ -382,16 +382,14 @@ static void vsetc(CType *type, int r, CValue *vc)
...
 	vsetc (type, VT_CONST, &cval);
 }
 
 /* push integer constant */
 ST_FUNC void vpushi(int v)
 {
-	CValue cval = {
-		0
-	};
+	CValue cval = { 0 };
```
Comment (XVilka) [inline shlr/tcc/tccgen.c]: please be consistent, no spaces here.

File: shlr/tcc/tccgen.c [hunk 2]
```diff
@@ -1792,8 +1790,7 @@ ST_FUNC void indir(void)
 
 /* parse an expression of the form '(type)' or '(expr)' and return its
    type */
-static void parse_expr_type(CType *type)
-{
+static void parse_expr_type(CType *type) {
```
Comment (XVilka) [inline shlr/tcc/tccgen.c]: Also inconsistency here - why do you need to have `{` at the same line?

File: libr/crypto/p/crypto_rc2.c [hunk 3]
```diff
@@ -158,7 +158,7 @@ static void rc2_dcrypt(struct rc2_state *state, const ut8 *inbuf, ut8 *outbuf, i
 		idx += 1;
 		if (idx % BLOCK_SIZE == 0) {
 			rc2_dcrypt8 (state, (const ut8 *) data_block, (ut8 *) dcrypted_block);
-			strncpy (ptr, dcrypted_block, BLOCK_SIZE);
+			memcpy (ptr, dcrypted_block, BLOCK_SIZE);
```
[the remark this answers is not in the document]
Comment (radare) [reply libr/crypto/p/crypto_rc2.c]: its raw data, the ending zero shouldnt matter at all

File: shlr/tcc/tccgen.c (hunk 1 above)
Comment (radare) [reply shlr/tcc/tccgen.c]: why no spaces? it should have, use r2i

File: shlr/tcc/tccgen.c (hunk 2 above)
Comment (radare) [reply shlr/tcc/tccgen.c]: because this is not TCC anymore, so it should follow our coding style

**Comment by radare** [comment]
Didnt wanted to introduce indent changes in a bugfixing pr but i can reindent the whole file right before mergint if you want

**Pull request merged**